packages/python/src/version.ts

```diff
@@ -0,0 +1,94 @@
+import { NowBuildError } from '@vercel/build-utils';
+
+interface PythonVersion {
+  version: string;
+  pipPath: string;
+  pythonPath: string;
+  runtime: string;
+  discontinueDate?: Date;
+}
+
+const allOptions: PythonVersion[] = [
```

Comment:
Should this have a comment that explains the the order is important? It seems that the first element needs to be the most recent version.